suite-native/message-system/src/components/MessageSystemRenderer.tsx

```diff
@@ -1,39 +1,52 @@
+import { ReactNode } from 'react';
 import { useSafeAreaInsets } from 'react-native-safe-area-context';
 import { useSelector } from 'react-redux';
 
 import { A } from '@mobily/ts-belt';
 
 import { Box, VStack } from '@suite-native/atoms';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { selectActiveBannerMessages } from '@suite-common/message-system';
+import {
+    selectActiveBannerMessages,
+    selectActiveFeatureMessages,
+} from '@suite-common/message-system';
 
 import { MessageBanner } from './MessageBanner';
+import { FeatureMessageScreen } from './FeatureMessageScreen';
 
 const messageSystemRendererStyle = prepareNativeStyle<{ topSafeAreaInset: number }>(
     (_, { topSafeAreaInset }) => ({
         marginTop: topSafeAreaInset,
     }),
 );
 
-export const MessageSystemRenderer = () => {
+type MessageSystemRendererProps = {
+    children: ReactNode;
+};
+
+export const MessageSystemRenderer = ({ children }: MessageSystemRendererProps) => {
     const { applyStyle } = useNativeStyles();
     const { top: topSafeAreaInset } = useSafeAreaInsets();
 
     const activeBannerMessages = useSelector(selectActiveBannerMessages);
-
-    if (A.isEmpty(activeBannerMessages)) return null;
+    const activeFeatureMessages = useSelector(selectActiveFeatureMessages);
+    const firstFeatureMessage = A.head(activeFeatureMessages);
 
     return (
-        <Box
-            style={applyStyle(messageSystemRendererStyle, {
-                topSafeAreaInset,
-            })}
-        >
-            <VStack spacing={4}>
-                {activeBannerMessages.map(message => (
-                    <MessageBanner key={message.id} message={message} />
-                ))}
-            </VStack>
-        </Box>
+        <>
+            {children}
+            <Box
+                style={applyStyle(messageSystemRendererStyle, {
```

Comment:
Could you please change name of  this style to something like `messageBannerContainerStyle` please? `messageSystemRendererStyle` does not make sense anymore.

Reply:
9860215781